spring-petclinic-microservices/pom.xml

```diff
@@ -33,7 +33,7 @@
     <docker.image.dockerfile.dir>${basedir}</docker.image.dockerfile.dir>
     <docker.image.dockerize.version>v0.6.1</docker.image.dockerize.version>
     <docker.plugin.version>1.2.2</docker.plugin.version>
-    <spring-boot-admin.version>2.7.10</spring-boot-admin.version>
+    <spring-boot-admin.version>3.0.0</spring-boot-admin.version>
```

Comment:
Hi, @backwind1233.
1. Does this sample support both spring-boot:2 and spring-boot:3?
2. How about moving these properties to parent pom, it's using `azure-spring-boot-samples` as parent.